apps/web/app/(app)/environments/[environmentId]/surveys/[surveyId]/(analysis)/summary/components/SummaryPage.tsx

```diff
@@ -155,26 +156,28 @@ export const SummaryPage = ({
   }, [searchParams, resetState]);
 
   return (
-    <>
-      <SummaryMetadata
-        surveySummary={surveySummary.meta}
-        showDropOffs={showDropOffs}
-        setShowDropOffs={setShowDropOffs}
-        isLoading={isLoading}
-      />
-      {showDropOffs && <SummaryDropOffs dropOff={surveySummary.dropOff} />}
-      <div className="flex gap-1.5">
-        <CustomFilter survey={surveyMemoized} />
-        {!isSharingPage && <ResultsShareButton survey={surveyMemoized} webAppUrl={webAppUrl} />}
-      </div>
-      <SummaryList
-        summary={surveySummary.summary}
-        responseCount={responseCount}
-        survey={surveyMemoized}
-        environment={environment}
-        totalResponseCount={totalResponseCount}
-        attributeClasses={attributeClasses}
-      />
-    </>
-  );
+    <ScrollToTopUi>
```

Comment:
Why is this a wrapper component. This seems unusual and we should have this as a non-wrapper `<ScrollToTopButton />` component.